osc/git_scm/store.py

```diff
@@ -31,8 +31,15 @@ def __init__(self, path, check=True):
         self.abspath = os.path.abspath(self.path)
 
         # TODO: how to determine if the current git repo contains a project or a package?
-        self.is_project = False
-        self.is_package = os.path.exists(os.path.join(self.abspath, ".git"))
+
+        # NOTE: we have only one store in project-git for all packages
+        self._toplevel = self._run_git(["rev-parse", "--show-toplevel"])
+        if self._toplevel == self.abspath:
+            self.is_project = True
+            self.is_package = False
+        else:
+            self.is_project = False
+            self.is_package = True
```

Comment:
I believe this is the change that broke unit tests.
See `tests / unit - read only fixtures (pull_request) `

Reply:
I am unable to find that check/failure, however I fixed a different one when executing it local.

There are still two test case failures, but IMHO the test cases are to blame here: tests/test_git_scm_store.py sets up the the work area with plain git (not osc co) and fails over missing .osc/_project later one.

IMHO this can not work, we either should set it up using "osc co" or also setup the .osc meta data.